packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/dto/CollapseExpandDiagramElementSuccessPayload.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.diagrams.dto;
+
+import java.text.MessageFormat;
+import java.util.Objects;
+import java.util.UUID;
+
+import org.eclipse.sirius.components.core.api.IPayload;
+
+/**
+ * The payload of the "collapse expand diagram element" mutation returned on success.
+ *
+ * @author gcoutable
+ */
+public class CollapseExpandDiagramElementSuccessPayload implements IPayload {
```

Comment:
This can now be replaced by the generic `SuccessPayload`. See https://github.com/eclipse-sirius/sirius-components/commit/23086a2757958949ed9984c45c0c2e97791a602e.

Reply:
Done